lib/improver/tests/psychrometric_calculations/psychrometric_calculations/test_FallingSnowLevel.py

```diff
@@ -39,10 +39,9 @@
 
 from improver.psychrometric_calculations.psychrometric_calculations import (
     FallingSnowLevel)
-from improver.tests.ensemble_calibration.ensemble_calibration.\
-    helper_functions import set_up_cube
-from improver.tests.utilities.test_mathematical_operations import (
-    set_up_height_cube)
+
```

Comment:
Just as a general comment, I notice that the `test_FallingSnowLevel.py` tests do generate FutureWarning messages that seem to come from the use of `stratify.interpolate`. Should we be managing these warnings in these unit tests?

Reply:
I'm loathed to hide future warnings as we need to deal with them, though in this case they are caused by an external package (which Bill Little maintains incidentally). We could raise an issue or a PR to fix the issue in stratify instead?

Reply:
What's happening about this?

Reply:
I checked out a local copy of stratify to have a look. The same warning is raised within their unit tests. The code was quite recently edited, so I assume Bill/Laura/others are still keeping it active. As such I was going to try and identify the source of the warning so I could raise an informed issue or PR. I've not got to this yet.

Reply:
I have raised a PR with stratify to remove the future warnings that we have chosen not to suppress: https://github.com/SciTools-incubator/python-stratify/pull/30